### PR TITLE
Chapter4 - Added PHP7 type declarations and added comments to methods  &variables

### PR DESCRIPTION
**Pull request by LoganConnor44**

### CODE REVIEW

**Comment by LoganConnor44** [comment]
Please take a moment at your earliest convenience to review the passing branches. If there is something you would like changed, let me know.

File: src/TDD/Chapter4/Dollar.php [hunk 1]
```diff
@@ -4,19 +4,42 @@
 
 class Dollar
 {
+
+	/**
```
Comment (guhelski) [inline src/TDD/Chapter4/Dollar.php]: Extra space.

File: src/TDD/Chapter4/Dollar.php [hunk 2]
```diff
@@ -4,19 +4,42 @@
...
+     * @var int $amount The monetary amount given to the dollar object
+     */
 	private $amount;
 
-	public function __construct($amount)
+	/**
```
Comment (guhelski) [inline src/TDD/Chapter4/Dollar.php]: Same here.

File: src/TDD/Chapter4/Dollar.php [hunk 3]
```diff
@@ -4,19 +4,42 @@
...
+     * 
+     * @param int $amount
+     */
+	public function __construct(int $amount)
 	{
 		$this->amount = $amount;
 	}
 
-	public function times($multiplier)
+	/**
```
Comment (guhelski) [inline src/TDD/Chapter4/Dollar.php]: Same here.

File: src/TDD/Chapter4/Dollar.php [hunk 4]
```diff
@@ -4,19 +4,42 @@
...
+     * @param int $multiplier
+     * @return object Dollar
+     */
+	public function times(int $multiplier): Dollar
 	{
 		return new Dollar($this->amount * $multiplier); 
 	}
 
-	public function equals(Dollar $object)
+	/**
```
Comment (guhelski) [inline src/TDD/Chapter4/Dollar.php]: Same here.

**Review by guhelski — CHANGES_REQUESTED** [review]
Minor formatting issues, extra spaces in the docblocks.

**Comment by LoganConnor44** [comment]
It seems that Sublime's indent / spaces does not play nicely with github.

The spacing issues have been fixed for this branch.

**Pull request merged**